server/src/domain/sql_backend_handler.rs

```diff
@@ -18,6 +18,12 @@ impl SqlBackendHandler {
 #[async_trait]
 impl BackendHandler for SqlBackendHandler {}
 
+impl WithContextHandler for SqlBackendHandler {
+    fn with_context(&self, _context: RequestContext) -> Self {
+        self.clone()
```

Comment:
Note on cloning the handler: the DB connection is just a handle so can be cloned cheaply. It also takes a `Config`, which I guess is acceptable to clone (at least for now), but could be refactored as an `Arc` instead, or just the necessary fields extracted.

Reply:
Marking this one as resolved, as we decided to pass the context, instead of using this trait.